internal/core/graphics.rs

```diff
@@ -187,6 +187,9 @@ pub enum RequestedGraphicsAPI {
     Vulkan,
     /// Direct 3D
     Direct3D,
+    #[cfg(feature = "unstable-wgpu-24")]
+    /// WGPU 24.x
+    WGPU24,
```

Comment:
If you make a different module, I think it could be to move the whole graphics selection API in core/api/graphics_selection.rs or something like that